### PR TITLE
Add support for NSIS 3.03

### PR DESCRIPTION
**Pull request by idleberg**
- add NSIS 3.03 commands
- fix: add missing separator between `!getdllversion` and `!system`
- update & alphabetize test

### CODE REVIEW

**Comment by Golmote** [comment]
Thanks and sorry for the all the confusion in #1285 ^^
Can you please remove `yarn.lock` from this PR (because it is unrelated to the NSIS changes) and run `gulp` to generate the minified version of the component?

**Comment by idleberg** [comment]
> Can you please remove yarn.lock from this PR

Whoops, sorry about that

File: tests/languages/nsis/important_feature.test [hunk 1]
```diff
@@ -22,9 +23,9 @@
 !macro
 !makensis
 !packhdr
-!pragma
```
Comment (Golmote) [inline tests/languages/nsis/important_feature.test]: I think you removed this test by mistake.

File: tests/languages/nsis/important_feature.test [hunk 2]
```diff
@@ -57,9 +59,9 @@
 	["important", "!macro"],
 	["important", "!makensis"],
 	["important", "!packhdr"],
-	["important", "!pragma"],
```
Comment (Golmote) [inline tests/languages/nsis/important_feature.test]: Same here.

File: tests/languages/nsis/keyword_feature.test [hunk 3]
```diff
@@ -41,7 +41,6 @@ EnumRegValue
 Exch
 Exec
 ExecShell
-ExecShellWait
```
Comment (Golmote) [inline tests/languages/nsis/keyword_feature.test]: Same here. This keyword still exists in the pattern, so it should be tested.

File: tests/languages/nsis/keyword_feature.test [hunk 4]
```diff
@@ -207,8 +213,6 @@ WriteINIStr
 WriteRegBin
 WriteRegDWORD
 WriteRegExpandStr
-WriteRegMultiStr
-WriteRegNone
```
Comment (Golmote) [inline tests/languages/nsis/keyword_feature.test]: Same here.

File: tests/languages/nsis/keyword_feature.test [hunk 5]
```diff
@@ -259,7 +263,6 @@ XPStyle
 	["keyword", "Exch"],
 	["keyword", "Exec"],
 	["keyword", "ExecShell"],
-	["keyword", "ExecShellWait"],
```
Comment (Golmote) [inline tests/languages/nsis/keyword_feature.test]: Same here.

File: tests/languages/nsis/keyword_feature.test [hunk 6]
```diff
@@ -425,8 +435,6 @@ XPStyle
 	["keyword", "WriteRegBin"],
 	["keyword", "WriteRegDWORD"],
 	["keyword", "WriteRegExpandStr"],
-	["keyword", "WriteRegMultiStr"],
-	["keyword", "WriteRegNone"],
```
Comment (Golmote) [inline tests/languages/nsis/keyword_feature.test]: Same here.

**Comment by Golmote** [comment]
Great, LGTM! Thanks for contributing!

**Pull request merged**